partials/_floating.scss

```diff
@@ -0,0 +1,142 @@
+.floating {
+    margin-top: 52px;
+    width: 100%;
+    display: flexbox;
+    flex-direction: column;
+    position: relative;
+    margin: auto;
+    
+
+    @include media ( ">=md" ) {
+        text-align: center;
+        padding: 0;
+        max-width: 768px;
+    }
+
+    @include media ( ">xl" ) {
+        max-width: 1024px;
+    }
+
+    &__block {
+        flex-direction: row;
+        max-height: fit-content;
+        margin-top: 150px;
+        height: fit-content;
+        width: 100%;
+        text-align: right;
+        
+        @include media ( ">md" ) {
+            margin-top: 0px;
+        }
+    }   
+
+    &__title {
+        color: $tws_coral;
+        font-family: $font-handwritten;
+        transform: rotate(-90deg);
+        font-weight: 400;
+        line-height: 49px;
+        position: relative;
+        font-size: 37.3px;
+        z-index: 2;
+        margin: 0px 0px 0px;
+        padding: 0;
+        width: 20%;
+        white-space: nowrap;
+        top: 250px;
+
+        @media (min-width: 450px) {
+            font-size: 48px;
+        }
+    
+        @include media ( ">=md" ) {
+            line-height: 99px;
+            font-size: 54px;
+            left: -140px;
+            width: 50%;
+            top: 320px;
+        }
+
+        @include media ( ">=lg" ) {
+            top: 250px;
+            width: 50%;
+            font-size: 64px;
+            // font-size should be 90px here according to figma but looks better at 64px to keep text in line with image. Should I change?
+        }
+
+        @include media ( ">=xl" ) {
+            font-size: 80px;
+            width: 45%;
+            top: 350px;
+        }
+    }
+
+    &__img { 
+        right: 16px;
+        margin-right: 16px; 
+        position: relative;
+        width: 80%;
+        margin: auto;
+        -webkit-filter: drop-shadow(16px 16px 0px $rose);
+        filter: drop-shadow(16px 16px 0px $rose);
+
+        @include media ( ">=md" ) {
+            max-width: 792px;
+            width: 70%;
+            margin-right: 15%;
+        }
+    }
+}
+
+.textbox {
+    background-color: $seashell;
+    max-width: 100%;
+    text-align: center;
+    padding: 16px 24px 16px;
+    margin: 0px 0px;
+    position: relative;
+    top: -50px;
+    z-index: -1;
+
+    @include media ( ">=md" ) {
+        margin-left: 45%;
+        z-index: 2;
+        width: fit-content;
+        padding: 0;
+        top: -125px;
+        margin-right: 45px;
+    }
+
+    &__brand-name {
+        color: $tws_coral;
+        font-family: $font-regular;
+        text-transform: uppercase;
+        padding-top: 84px;
+    }
+
+    &__header {
+        font-family: $font-heading;
+        color: $tws_black;
+        font-weight: 300;
+        line-height: 40px;
+        font-size: 32px;
+        margin: 0px;
```

Comment:
I think that both heading styles and the red title above it could be extracted to some more general class (obviously, not all of them but the general ones e.g. color, font) - they are elements that are likely to be reused in various places.

Reply:
yeah, again - remember they can be both separate blocks (e.g. `.heading`) and elements of a different block (`.textbox__header`) at the same time if needed.

Reply:
Have renamed service__header and textbox__header to .heading and declared this within the general.scss file now instead.